pom.xml

```diff
@@ -0,0 +1,125 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.querydsl</groupId>
+  <artifactId>querydsl-parent</artifactId>
+  <version>0.1.0</version>
+  <name>Querydsl Parent POM</name>
+  <description>Parent project for Querydsl related projects</description>
+  
+  <parent>
+    <groupId>org.sonatype.oss</groupId>
+    <artifactId>oss-parent</artifactId>
+    <version>7</version>
+  </parent>  
+  
+  <url>http://www.github.com/querydsl/querydsl-parent</url>
+  <packaging>pom</packaging>
+
+  <inceptionYear>2015</inceptionYear>
+  
+  <organization>
+    <name>Querydsl</name>
+    <url>http://www.querydsl.com</url>
+  </organization>
+
+  <developers>
+    <developer>
+      <id>timowest</id>
+      <name>Timo Westkämper</name>
+      <organization>Mysema Ltd</organization>

```

Comment:
@timowest Do we make the Querydsl Team members organization neutral? Or do you want this included?